### PR TITLE
7080 smart search in talks page

### PR DESCRIPTION
**Pull request by kris45**

### CODE REVIEW

File: conference/react-js/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "node-sass": "^4.5.3",
     "optimize-css-assets-webpack-plugin": "^2.0.0",
     "react-addons-test-utils": "^15.6.0",
+    "react-input-select": "^1.0.1",
```
Comment (blyzniuk) [inline conference/react-js/package.json]: two times?

File: conference/react-js/src/containers/Talks/FilterForm/index.jsx [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      topicValue: '',
+      listOfTopics: [],
+    };
+  }
+
+  componentWillReceiveProps() {
+    const { topics } = this.props;
```
Comment (blyzniuk) [inline conference/react-js/src/containers/Talks/FilterForm/index.jsx]: Should you do this with new props or with current? 
```
componentWillReceiveProps(newProps) {
   this.props // === old props
}
```

File: conference/react-js/src/containers/Talks/FilterForm/index.jsx [hunk 3]
```diff
@@ -20,14 +53,14 @@ class FilterForm extends PureComponent {
             <label
               htmlFor="my-talk-status"
               className="form-label my-talk-settings__label
-              my-talk-settings__label_status"
```
Comment (blyzniuk) [inline conference/react-js/src/containers/Talks/FilterForm/index.jsx]: As for me, the previous indentation was better

File: conference/react-js/src/containers/Talks/FilterForm/index.jsx [hunk 4]
```diff
@@ -11,8 +11,8 @@ class FilterForm extends PureComponent {
     };
   }
 
-  componentWillReceiveProps() {
-    const { topics } = this.props;
+  componentWillReceiveProps(nextProps) {
+    const { topics } = nextProps;
     const listOfTopics = [];
```
Comment (blyzniuk) [inline conference/react-js/src/containers/Talks/FilterForm/index.jsx]: const listOfTopics = topics.map(({ name }) => name)

**Review by blyzniuk — APPROVED** [review]

**Review by ScreamCat1 — APPROVED** [review]

**Pull request merged**